src/QMCHamiltonians/Listener.hpp

```diff
@@ -18,6 +18,7 @@
  *  implementation of observables that rely on per particle Hamiltonian values.
  */
 
+#include <complex>
 #include <functional>
```

Comment:
Could you remove the `OhmmsPETE/OhmmsMatrix.h` include below. Probably also add include <vector>